app/packages/grits-net-meteor/client/templates/grits_search.coffee

```diff
@@ -10,6 +10,7 @@ _effectiveDatePicker = null # onRendered will set this to a datetime picker obje
 _discontinuedDatePicker = null # onRendered will set this to a datetime picker object
 _matchSkip = null # the amount to skip during typeahead pagination
 _simulationProgress = new ReactiveVar(0)
+@notify = new ReactiveVar(false)
```

Comment:
We should probably declare this on the template instance in `onCreated` if possible.
Then we can avoid the global variable and using the double arrow in defining the related helper and event. (Then access with `Template.instance()`)